components/gcp-click-to-deploy/src/DeployForm.tsx

```diff
@@ -14,31 +14,33 @@ import * as React from 'react';
 import * as request from 'request';
 
 import Gapi from './Gapi';
-import {encryptPassword, flattenDeploymentOperationError, getFileContentsFromGitHub, log, wait} from './Utils';
+import {
+    encryptPassword, flattenDeploymentOperationError,
+    getFileContentsFromGitHub, log, wait
+} from './Utils';
 
 /** Relative paths from the root of the repository. */
 enum ConfigPath {
     V05 = 'v0.5-branch/components/gcp-click-to-deploy/app-config.yaml',
-    V06 = 'v0.6-branch/bootstrap/config/kfctl_gcp_iap.0.6.yaml'
+    V06 = 'master/bootstrap/config/kfctl_gcp_iap.yaml'
```

Comment:
`v0.6-branch` should be more stable compared to `master`
`master` branch could be broken at any time.

Reply:
Yes I agree. But [v0.6-branch/bootstrap/config/kfctl_gcp_iap.0.6.yaml](https://github.com/kubeflow/kubeflow/blob/v0.6-branch/bootstrap/config/kfctl_gcp_iap.0.6.yaml)  doesn't have the `applications` section set. [master/bootstrap/config/kfctl_gcp_iap.yaml](https://github.com/kubeflow/kubeflow/blob/master/bootstrap/config/kfctl_gcp_iap.yaml) has `applications` but not `components`, `componentParams`, or `packages`.

If possible, we should check a base config file into the v0.6-branch that has the main structure that needs to be in the request body so that the UI doesn't have to build any new elements but simply replace parameter values. Alternatively, we could revert to the mechanism used in the current deployment UI where we provide the config file at deployment time, but pulling from GitHub definitely reduces the deployment complexity.